opendevin/sandbox/docker/exec_box.py

```diff
@@ -61,7 +62,7 @@ def __init__(
             raise ex
 
         self.instance_id = sid if sid is not None else str(uuid.uuid4())
-
+        self.workspace_mount_path = workspace_mount_path or config.get(ConfigType.WORKSPACE_MOUNT_PATH)
```

Comment:
So this isn't quite the logic I think we should have.

Instead, `WORKSPACE_MOUNT_PATH` should be held constant, and the sandboxes should accept a _relative_ directory, `like `./app`